cli/src/transport.ts

```diff
@@ -37,22 +38,22 @@ function tryRequire(name: string) {
     return require(name)
 }
 
-function createSPI() {
-    log(`adding SPI transport (requires "rpio" package)`)
+async function createSPI() {
+    await askForPackageInstallation('rpio');
```

Comment:
We need a non-interactive mode in CI build mode.

Reply:
Good point! Flag added globally to the cli in this commit https://github.com/microsoft/devicescript/pull/653/commits/ddf9c15d7f374d1a93745e3eabc5dfaac6702f54